configure-local-networking.sh

```diff
@@ -31,19 +33,43 @@ if $(which dnf >/dev/null 2>&1); then
     sudo dnf -y install iptables
 fi
 
+if [[ $ID = "ubuntu" ]]; then
+    sudo apt update
+    sudo apt -y install ifupdown
+fi
+
 # Configure local networking.
 # Add a bridge 'braio' for the Kayobe all-in-one cloud network.
 if ! sudo ip l show braio >/dev/null 2>&1; then
   sudo ip l add braio type bridge
-  sudo ip l set braio up
-  sudo ip a add $seed_hv_ip/24 dev braio
+  if [[ $ID = "ubuntu" ]]; then
+    cat << EOF | sudo tee /etc/network/interfaces.d/ifcfg-braio
```

Comment:
If I understand the Ansible interfaces role correctly, this file will be overwritten by Kayobe during seed hypervisor host configuration. It would be nice to make it so that running `configure-local-networking.sh` another time (as you are supposed to do after rebooting) doesn't change the file again.

Reply:
It should have dumb idempotency due to `if ! sudo ip l show braio >/dev/null 2>&1; then`

Reply:
Right, I wasn't looking at enough context. Thanks.